src/Tribe/Admin/Settings.php

```diff
@@ -385,6 +387,18 @@ public function settings_ui( $admin_page ) {
 		$this->tabs['general'] = new Tribe__Settings_Tab( 'general', esc_html__( 'General', 'the-events-calendar' ), $general_tab );
 		$this->tabs['display'] = new Tribe__Settings_Tab( 'display', esc_html__( 'Display', 'the-events-calendar' ), $tec_events_display_tab );
 		add_filter( 'tribe_settings_tabs', [ $this, 'sort_tabs' ], 100, 2 );
+
+		add_action(
+			'tribe_settings_after_form_div',
+			function() {
```

Comment:
```suggestion
			function () {
```

Reply:
Honestly though, I think this should live in the parent tab - as each will have a different one - or may not have one? I can be convinced otherwise...